src/swoop/api/routers/processes.py

```diff
@@ -72,16 +84,19 @@ async def list_processes(
     workflows = (
         request.app.state.workflows.get("workflows")
         if request.app.state.workflows.get("workflows")
-        else []
+        else {}
     )
 
+    workflows = list(workflows.values())
+
     if queryparams and len(workflows) > 0:
         workflows = list(
             filter(
-                lambda x: queryparams.items() <= x.items(),
-                request.app.state.workflows["workflows"],
+                lambda x: queryparams.items() <= vars(x).items(),
+                workflows,
             )
         )
+
```

Comment:
I'm not sure if I see the reason to filter workflows like this. Maybe we can discuss in parking lot whether this is valuable and warrants the additional complexity.